Framework/Examples/LorentzianTest.cpp

```diff
@@ -72,20 +72,17 @@ void LorentzianTest::functionLocal(double *out, const double *xValues,
 // I.e. substitute the code below with the four lines of code above
 void LorentzianTest::functionDerivLocal(Jacobian *out, const double *xValues,
                                         const size_t nData) {
-  const double &height = getParameter("Height");
-  const double &peakCentre = getParameter("PeakCentre");
-  const double &hwhm = getParameter("HWHM");
+  const double h = height();
+  const double c = centre();
+  const double w = fwhm() / 2.;
 
   for (size_t i = 0; i < nData; i++) {
-    double diff = xValues[i] - peakCentre;
-    double invDenominator = 1 / ((diff * diff + hwhm * hwhm));
-    out->set(i, 0, hwhm * hwhm * invDenominator);
-    out->set(i, 1,
-             2.0 * height * diff * hwhm * hwhm * invDenominator *
-                 invDenominator);
+    double diff = xValues[i] - c;
+    double invDenominator = 1 / ((diff * diff + w * w));
```

Comment:
Can `diff` and `invDenominator` be `const`?

Reply:
Yes they can.